### PR TITLE
Check error on loading config, don't print if missing config file

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Fixes https://github.com/giantswarm/builder/issues/82

### CODE REVIEW

**Comment by JosephSalisbury** [comment]
As a note, I tested this with config files being both present and absent, incorrect formats, for both json and yml.

Regardless of whether the file was present or not, no message was printed. When the config file was incorrect (garbage data, say), an error message was printed.

**Comment by JosephSalisbury** [comment]
@giantswarm/cena RFR

**Comment by xh3b4sd** [comment]
Is there a way to test this automatically?

**Comment by JosephSalisbury** [comment]
Not certain - it's mucky with the filesystem, and viper over it as well

**Comment by xh3b4sd** [comment]
I see. Lets go with this. LGTM

File: commands/builder.go [hunk 1]
```diff
@@ -35,7 +35,12 @@ func init() {
 	viper.AddConfigPath(".")
 
 	if err := viper.ReadInConfig(); err != nil {
-		fmt.Printf("Error occurred reading config: %v, continuing\n", err)
+		// If the error is not the config file not being found, print a message.
+		// Note: viper apparently returns UnsupportedConfigError,
+		// not ConfigFileNotFoundError when the config file cannot be found.
```
Comment (zeisss) [inline commands/builder.go]: do we want to create an issue for this upstream?

**Comment by zeisss** [comment]
LGTM

File: commands/builder.go (hunk 1 above)
Comment (JosephSalisbury) [inline commands/builder.go]: I'll put it on my todo list to look into it properly.

**Pull request merged**